### PR TITLE
doc: add README for toolchain

### PR DESCRIPTION
**Pull request by imbajin**
after merge to apache, we need do

- [ ] update all urls in `README.md`
- [ ] update some `loader` link to `toolchains`
- [ ] we should better unify the version for loader/hubble/tools with the server to keep consistency

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# hugegraph-toolchains
```
Comment (javeme) [inline README.md:1]: prefer to use 'hugegraph-toolchain' name, since the repo name is hugegraph-toolchain

File: README.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+[![License](https://img.shields.io/badge/license-Apache%202-0E78BA.svg)](https://www.apache.org/licenses/LICENSE-2.0.html)
+[![Build Status](https://github.com/hugegraph/hugegraph-loader/actions/workflows/ci.yml/badge.svg)](https://github.com/hugegraph/hugegraph-loader/actions/workflows/ci.yml)
+[![codecov](https://codecov.io/gh/hugegraph/hugegraph-loader/branch/master/graph/badge.svg)](https://codecov.io/gh/hugegraph/hugegraph-loader)
+[![Maven Central](https://maven-badges.herokuapp.com/maven-central/com.baidu.hugegraph/hugegraph-loader/badge.svg)](https://mvnrepository.com/artifact/com.baidu.hugegraph/hugegraph-loader)
+
+`hugegraph-toolchains` is the integration of a series of utilities for [HugeGraph](https://github.com/hugegraph/hugegraph), it includes 4 main modules.
```
Comment (javeme) [inline README.md:8]: "... is the integration project of ..."

File: README.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+## Modules
+
+- [hugegraph-loader](./hugegraph-loader): Loading datasets into the HugeGraph from multiple data sources.
+- [hugegraph-hubble](./hugegraph-hubble): Online HugeGraph management and analysis dashboard (Include: data loading, schema management, graph traverser and display)
+- [hugegraph-tools](./hugegraph-tools): Command line tool for deploying, managing and backing up/restoring graphs from HugeGraph.
```
Comment (javeme) [inline README.md:14]: backing-up

File: README.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+- [hugegraph-client](./hugegraph-client): A Java-written client for HugeGraph, providing RESTful graph APIs for schema/gremlin/variables and traversals etc.
```
Comment (javeme) [inline README.md:15]: providing RESTful APIs for accessing graph vertex/edge/schema/gremlin/variables and traversals etc.

File: README.md [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+
+## Doc
+
+The [project homepage](https://hugegraph.github.io/hugegraph-doc/) contains more information about `hugegraph-toolchains`.
```
Comment (javeme) [inline README.md:19]: dtto

**Review by javeme — APPROVED** [review]

**Review by corgiboygsj — APPROVED** [review]

**Pull request merged**